doc/classes/Node3D.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Node3D" inherits="Node" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Most basic 3D game object, parent of all 3D-related nodes.
+		Base object in 3D space, inherited by all 3D nodes.
 	</brief_description>
 	<description>
-		Most basic 3D game object, with a [Transform3D] and visibility settings. All other 3D game objects inherit from [Node3D]. Use [Node3D] as a parent node to move, scale, rotate and show/hide children in a 3D project.
-		Affine operations (rotate, scale, translate) happen in parent's local coordinate system, unless the [Node3D] object is set as top-level. Affine operations in this coordinate system correspond to direct affine operations on the [Node3D]'s transform. The word local below refers to this coordinate system. The coordinate system that is attached to the [Node3D] object itself is referred to as object-local coordinate system.
-		[b]Note:[/b] Unless otherwise specified, all methods that have angle parameters must have angles specified as [i]radians[/i]. To convert degrees to radians, use [method @GlobalScope.deg_to_rad].
-		[b]Note:[/b] Be aware that "Spatial" nodes are now called "Node3D" starting with Godot 4. Any Godot 3.x references to "Spatial" nodes refer to "Node3D" in Godot 4.
+		The [Node3D] node is the base representation of a node in 3D space. All other 3D nodes inherit from this class.
+		Affine operations (translation, rotation, scale) are calculated in the coordinate system relative to the parent, unless the [Node3D]'s [member top_level] is [code]true[/code]. In this coordinate system, affine operations correspond to direct affine operations on the [Node3D]'s [member transform]. The term "[i]parent space[/i]" refers to this coordinate system. The coordinate system that is attached to the [Node3D] itself is referred to as object-local coordinate system, or "[i]local space[/i]".
```

Comment:
What's the reasoning for the double emphasis for new technical terms? IMO either italics or quotes is sufficient. Both seems overkill.

Reply:
Not much reasoning other than personal preference. In this particular case it was to make these words stand out even more, because they are referenced in the rest of the class reference without this explanation, but it's not... necessary.

Reply:
This is your call since the class reference is your domain, but I personally think it's too much emphasis. As one more point of data, I don't believe it's currently used. Searching for `"[i]` and `[i]"` finds 0 and 1 current instances (and the second one is using italics to reference an error message, not for new term introduction as you are doing here).

Reply:
Nah, you're right, it's not a good precedent.